resources/views/home.blade.php

```diff
@@ -0,0 +1,64 @@
+@extends('layouts.app')
+
+@section('content')
+
+<!DOCTYPE html>
```

Comment:
You are already extending `app.blade.php` by the @extends('layouts.app') so no need to put `<!DOCTYPE html>` and all other code found in `app.blade.php` here since it is already in `app.blade.php.

You can directly put your content under @section('content'). Like this:
```
@section('content')
@foreach($users as $user)
    <div class="row">
      //.... add remaining code here
     ...
     ...
@endsection
```